modules/arm_plugin/src/arm_converter/arm_converter_pool.cpp

```diff
@@ -69,6 +70,16 @@ template<> Converter::Conversion::Ptr Converter::Convert(const ngraph::op::v8::M
         FillLayerInfo(node, pool_info);
         pool_info.pool_type = arm_compute::PoolingType::MAX;
         return MakeConversion<arm_compute::NEPoolingLayer>(node.input(0), node.output(0), pool_info, node.output(1));
+    } else if ((node.get_input_shape(0).size() == 4) &&
```

Comment:
In this case also possible to use transformation for `ngraph::op::v8::MaxPool` like `ArmPlugin::pass::ConvertMaxPool1D::ConvertMaxPool1D()` to process 3D input shapes too

Reply:
I've replaced special processing with V8->V1 transformation. Now all possible cases should be handled if they are supported by V1